README.md

```diff
@@ -457,7 +457,12 @@ Instead of using `dotnet publish` to deploy changes to a database, you can also
 </Project>
 ```
 
-With this enabled you'll find a SQL script with the name `<datbase-name>_Create.sql` in the bin folder.
+With this enabled you'll find a SQL script with the name `<database-name>_Create.sql` in the bin folder.
+The database name for the create script gets resolved in the following manner:
+1. `TargetDatabaseName`.
+1. Package name.
```

Comment:
Should it be 1, 2, 3?

Reply:
It displays as 1, 2, 3... Don't know what it would add?

1. 1
1. 2
1. 3

vs

1. 1
2. 2
3. 3